spec/property.dd

```diff
@@ -4,7 +4,7 @@ $(SPEC_S Properties,
 
 $(HEADERNAV_TOC)
 
-        $(P Every type and expression has properties that can be queried:)
+        $(P Every symbol,type and expression has properties that can be queried:)
```

Comment:
space after comma

Reply:
@JackStouffer do you have rights to edit yet?

Reply:
@andralex I tried in the past to edit other people's PRs, but the problem I ran into is it seems to only work over `git://` and not `https://`, so while I'm at work the firewall stops me.